src/core_atmosphere/physics/physics_noaa/SMOKE/module_fire_emissions.F90

```diff
@@ -0,0 +1,68 @@
+!>\file  module_wildfire_smoke_emissions.F90
+!! This file contains the MPAS-Aerosols/RRFS wildfire emission module
+
+module module_wildfire_smoke_emissions
+!
+!  This module developed by Johana Romero-Alvarez and Jordan Schnell (NOAA GSL)
+!  For serious questions contact johana.romero-alvarez@noaa.gov
+!
+  use mpas_kind_types
+  use mpas_smoke_init, only : p_smoke_fine, p_smoke_coarse
+  use mpas_smoke_config
+
+  implicit none
+
+  private
+
+  public :: caclulate_smoke_emissions
```

Comment:
this is spelled differently than the subroutine

Reply:
The subroutine is a placeholder just for testing purpose.